trace_replay/block_cache_tracer.h

```diff
@@ -0,0 +1,91 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include "rocksdb/env.h"
+#include "rocksdb/options.h"
+#include "rocksdb/trace_reader_writer.h"
+#include "trace_replay/trace_replay.h"
+
+namespace rocksdb {
+
+enum BlockCacheLookupCaller : char {
+  kUserGet = 1,
+  kUserMGet = 2,
+  kUserIterator = 3,
+  kPrefetch = 4,
+  kCompaction = 5
+};
+
+enum Boolean : char { kTrue = 1, kFalse = 2 };
+
+struct TraceRecord {
```

Comment:
I feel this struct `TraceRecord` should have a more specific name, say, `BlockCacheTraceRecord`?

Reply:
Thanks. I renamed to BlockCacheTraceRecord.